lib/pluginmanager/uninstall.rb

```diff
@@ -1,13 +1,16 @@
 # encoding: utf-8
```

Comment:
Maybe we could make the uninstall inherits the remove command, override `#execute` and call super? This would make sure don't have any duplicated code around.